src/components/FileUploader/FileUploader.tsx

```diff
@@ -10,17 +11,23 @@ interface FileUploaderProps {
 }
 
 export const FileUploader: FC<FileUploaderProps> = ({uploadLink, acceptedFormats, refetch}) => {
+  const {mutate: fileUpload} = useMutation({
+    mutationFn: (formData: FormData) => {
+      return axios.post(uploadLink, formData)
+    },
+    onSuccess: () => refetch(),
+  })
+
   const onDrop = useCallback<NonNullable<DropzoneOptions['onDrop']>>(
     async (acceptedFiles, fileRejections) => {
       if (fileRejections.length > 0) {
         return
       }
       const formData = new FormData()
       formData.append('file', acceptedFiles[0])
-      await axios.post(uploadLink, formData)
-      await refetch()
+      fileUpload(formData)
```

Comment:
tu asi patri `await`, nie je dovod sa mu vyhybat

Reply:
nie je to promise, takze cela fcia nemusi byt ani async